apps/x509.c

```diff
@@ -745,7 +745,6 @@ int x509_main(int argc, char **argv)
 
     if (!noout || text || next_serial)
         OBJ_create("2.99999.3", "SET.ex3", "SET x509v3 extension 3");
-    /* TODO: why is this strange object created (and no error checked)? */
```

Comment:
This is still a strange object...

Reply:
Yup, and the comment just makes it stranger...

Reply:
SET is dead.  You could just move this to the regular objects files.

Reply:
The SET extension is "we know what the next serial# will be"  Part of the strange thing about the SET root CA.